english/ghostbusters/ghostbusters.md

```diff
@@ -12,13 +12,13 @@ and a strange tune begins forming in your mind...
 
 You immediately look around and notice a marshmallow phone.  You pick it up and dial the Ghostbusters.
 
-The air is suddenly filled with the sounds of blaring sirens as the Ghostbuster mobile 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.